Gems/AWSGameLift/Code/AWSGameLiftClient/Source/AWSGameLiftClientManager.cpp

```diff
@@ -349,6 +359,28 @@ namespace AWSGameLift
         return response;
     }
 
+    AZStd::string AWSGameLiftClientManager::StartMatchmaking(const AzFramework::StartMatchmakingRequest& startMatchmakingRequest)
+    {
+        AZ_UNUSED(startMatchmakingRequest);
+
+        return "";
```

Comment:
FYI returning `AZStd::string{}` is more efficient.
`""` needs to at least have `strlen` invoked on it to figure out it is zero bytes in lenght.
A default constructed string doesn't have to deal with that